packages/dev/core/src/Materials/Textures/thinTexture.ts

```diff
@@ -283,10 +283,11 @@ export class ThinTexture {
      *    > _min_: minification filter (far from the viewer)
      *    > _mip_: filter used between mip map levels
      *@param samplingMode Define the new sampling mode of the texture
+     *@param generateMipMaps Define if the texture should generate mip maps or not. Default is false.

```

Comment:
Is it worth clarifying that mip maps will still not be generated if this argument is true if the underlying texture does not have mip maps? Not sure if that would throw ppl off.